components/plugin/index.js

```diff
@@ -1,14 +1,13 @@
 import Highlighter from 'react-highlighter'
 import styles from './plugin.module.css'
+import Image from 'next/image'
 
 const MAX_COLORS = 4
 
 export default ({ featured, name, description, colors, preview, query }) =>
   featured ? (
-    <div
-      className={styles.featured}
-      style={{ backgroundImage: `url(${preview})` }}
-    >
+    <div className={styles.featured}>
+      <Image className={styles.featuredImage} src={preview.src} layout="fill" />
```

Comment:
We need to request the image in a specific, smaller, size. Right now the plugins page is loading 2.6mb+ images, which is not reasonable.

Reply:
Is there a way to do this through next/image?

Reply:
I tried forcing next to always load the 640px wide (smallest default device size) image by adding sizes="1px" (used 1px bc using the actual size of the image on the screen ~400px caused the browsers to pick a larger image due to retina screen), but the actual size of the gifs did not get smaller, maybe the _next/image endpoint is not that good at making gifs smaller?

Reply:
I could take a screenshot of those gifs and use that as preview images, or should i just make the resolution of the raw gif files smaller?

Reply:
Can we not run the gifs through `next/image` and just continue serving them via GitHub URLs?